docs/source/apis/aws_marketplace.rst

```diff
@@ -0,0 +1,245 @@
+
+AWS Marketplace Deployment Guide
+================================
+
+This guide provides step-by-step instructions for deploying Unstructured API from AWS Marketplace.
+
+Pre-Requirements
+----------------
+
+1. **AWS Account**: Register at `AWS Registration Page <https://aws.amazon.com/>`_, if you don't have an AWS account.
+
+2. **IAM Permissions**: Ensure permissions for ``CloudFormation``.
+
+   - Refer to this `AWS blog post <https://blog.awsfundamentals.com/aws-iam-roles-with-aws-cloudformation#heading-creating-iam-roles-with-aws-cloudformation>`_ to create IAM Roles with CloudFormation.
+
+3. **SSH KeyPair**: Create or use an existing KeyPair for secure access.
+
+   - Follow the ``Create Key Pairs`` in the Amazon EC2 `User Guide <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/create-key-pairs.html>`_.
+
+
+Part 1: Setting-Up a Virtual Private Cloud (VPC)
+------------------------------------------------
+
+1. **Access VPC Dashboard**:
+
+   - In the AWS Management Console, navigate to the VPC service.
+   - Click on “Your VPCs” in the left navigation pane, then “Create VPC.”
+
+2. **Create VPC**:
+
+   - Select ``VPC only``
+   - Enter a ``Name tag`` for your VPC.
+   - Specify the IPv4 CIDR block (e.g., 10.0.0.0/16).
+
+     - You may leave the IPv6 CIDR block and Tenancy settings as default.
+   - Click “Create VPC” button
+
+.. image:: imgs/AWS/VPC_Step2.png
+  :align: center
+  :alt: create vpc
+
+3. **Create Subnets**:
+
+   - After creating the VPC, click on “Subnets” in the left navigation pane.
+   - Click “Create subnet” and select the VPC you just created from the dropdown menu.
+   - For the first public subnet:
+
+     - Enter a ``Name tag``.
+     - Select an ``Availability Zone``.
+     - Specify the IPv4 CIDR block (e.g., 10.0.1.0/24).
```

Comment:
option here will be dependent oh what you put in before. maybe update to /16? since that's what you specified above?
```suggestion
     - Specify the IPv4 CIDR block (e.g., 10.0.1.0/16).
```